pom.xml

```diff
@@ -963,4 +971,37 @@
     </snapshotRepository>
   </distributionManagement>
 
+  <profiles>
+    <!-- Spark version profile -->
+    <profile>
+      <id>spark-1.6</id>
+      <activation>
+        <property>
+          <name>!spark-2.0</name>
+        </property>
+      </activation>
+      <properties>
+        <spark.version>1.6.2</spark.version>
+        <py4j.version>0.9</py4j.version>
+        <netty.version>4.0.23.Final</netty.version>
+        <json4s.version>3.2.10</json4s.version>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>spark-2.0</id>
```

Comment:
The json4s issue is unfortunate and one of the reasons why I'd like to get rid of that dependency... but Livy shouldn't be overriding Spark's version of netty. It may upload a netty jar with the app, but that should be in the app's class loader, not on the system class loader, so Spark should still use its own.

Anyway, it's fine for now until the issue of supporting multiple versions simultaneously is sorted out.